src/LpSlamNode.cpp

```diff
@@ -418,6 +434,11 @@ void LpSlamNode::image_callback_left(const sensor_msgs::msg::Image::SharedPtr ms
     if (camera_encoding.empty()) {
         setCameraEncoding(msg->encoding);
     }
+    
+    if (m_useRosCameraInfo && !m_cameraConfigured) {
```

Comment:
I am afraid that `m_cameraConfigured` could simultaneously be read and written from 2 different threads: `image_callback_left` (read) and `camera_info_callback` (written). May be, this should be protected by mutex, or `std::atomic_bool` to be used?

Reply:
Oh, right. Thank you for pointing that out!